templates/basics.md

```diff
@@ -30,21 +30,21 @@ I don't.
 No one does. 
 Incidentally, there are currently very few known lucrative diseases.  
 
-This site looks at what has happened.
+This site looks at what has happened using historical financial data [Note to Mike - I don't actually know where the data comes from? Like I guess it's from the stock market??].
```

Comment:
you can say, "using the same data as the Trinity Study" or just using historical stock and bond data. Its the SP500 and Ibbotson's Long Term Corporate Bond Index.